Lesson_5_Protractor/conf.js

```diff
@@ -0,0 +1,5 @@
+exports.config = {
+    framework: 'jasmine',
+    seleniumAddress: 'http://localhost:4444/wd/hub',
+    specs: ['spec.js']
+};
```

Comment:
Ernest jan where are the browser configurations? ))

Reply:
fixed